src/nix/hive/eval.nix

```diff
@@ -98,6 +98,8 @@ let
   in {
     options = {
       deployment = {
+        nixpkgs = lib.mkOption {
```

Comment:
Obviously, this would have to be fleshed out; this was just the minimal change I could make to get my test hive building.